framework/decode/custom_vulkan_ascii_consumer.h

```diff
@@ -241,11 +237,11 @@ inline std::string HandlePointerDecoderArrayToString(const CountType&
 
 /// Traverse arrays of simple types like uint32_t, not structs or handles.
 template <typename CountType, typename PointerDecoderType>
-inline std::string PointerDecoderArrayToString(const CountType&    countObj,
-                                               PointerDecoderType* pObjs,
-                                               util::ToStringFlags toStringFlags = util::kToString_Default,
-                                               uint32_t            tabCount      = 0,
-                                               uint32_t            tabSize       = 4)
+inline std::string PointerDecoderArrayToString(const CountType&          countObj,
+                                               const PointerDecoderType* pObjs,
```

Comment:
This const was added although not required for the change. It always should have been const: this is a non-modifying operation.